src/csrfHeader.js

```diff
@@ -4,15 +4,16 @@
  * @providesModule csrfHeader
  * @flow
  */
+import type { CustomHeaders } from './types'
 
-function getToken(): ?string {
+export function getToken(): ?string {
```

Comment:
Is this being used anywhere else? If not, it doesn't need to be exported

Reply:
no it's not. I was exporting it because the first way I wrote out the test was trying to be able to mock this function instead of the whole csrfHeader function, and then forgot to remove it